### ISSUE TITLE
Using Z shell (zsh) without oh-my-zsh

### ISSUE THREAD

**Issue by evgenTraytyak** [datarockets/dotfiles#1]
What you think about using `zsh` as default shell?
And use it together with [oh-my-zsh](http://ohmyz.sh/).

**Comment by dzhlobo** [comment]
I like zsh.

Not sure about oh-my-zsh. Do we really need it? Personally I use too little of its features. It requires separate updating.

**Comment by evgenTraytyak** [comment]
`.zshrc`

``` sh
#!/bin/zsh

# To enable the famous zsh tab-completion system
autoload -U compinit
compinit

# Enable the auto-correction of the commands typed
setopt correctall

# Initialize advanced prompt support
autoload -U promptinit
promptinit

# Let avoid tedious typing of cd command while changing current directory 
setopt autocd

# To prevent history from recording duplicated entries (such as ls -l entered many times during single shell session)
setopt hist_ignore_all_dups

# Prevent particular entries from being recorded into a history by preceding them with at least one space
setopt hist_ignore_space
```

**Comment by evgenTraytyak** [comment]
I suggest to add `archive` and `extract` function to work with archives.

**Comment by dzhlobo** [comment]
Resolved by merging #14.
